pyproject.toml

```diff
@@ -586,3 +797,56 @@ module = [
 ]
 check_untyped_defs = false
 disallow_untyped_defs = false
+
+
+[tool.coverage.report]
+exclude_lines = [
+    "pragma: no cover",
+    "if TYPE_CHECKING:",
+    "raise NotImplementedError()",
+    "except ImportError:",
+    "^ +\\.\\.\\.$",
+]
+
+[tool.coverage.run]
+parallel = true
+omit = [
+    "*/_vendor/*",
+    "*/_version.py",
+    "*/benchmarks/*",
+    "napari/utils/indexing.py",
+]
+source = [
+    "napari",
+    "napari_builtins",
+]
+
+[tool.coverage.paths]
+source = [
+    "napari/",
+    "D:\\a\\napari\\napari\\napari",
+    "/home/runner/work/napari/napari/napari",
+    "/Users/runner/work/napari/napari/napari",
+]
+builtins = "napari_builtins/\nD:\\a\\napari\\napari\\napari_builtins\n/home/runner/work/napari/napari/napari_builtins\n/Users/runner/work/napari/napari/napari_builtins"
+
+[tool.importlinter]
```

Comment:
Based on the docs: https://import-linter.readthedocs.io/en/stable/usage.html#top-level-configuration
The quotes and \n should be removed.

Reply:
@Czaki what's the status with this comment?

Reply:
Debugged, and fixed